test-common/test-common-api/src/main/java/org/apache/kafka/common/test/api/RaftClusterInvocationContext.java

```diff
@@ -175,8 +175,10 @@ public KafkaClusterTestKit getUnderlying() {
         }
 
         @Override
-        public Admin createAdminClient(Properties configOverrides) {
-            Admin admin = Admin.create(clusterTestKit.newClientPropertiesBuilder(configOverrides).build());
+        public Admin admin(Map<String, Object> overrides) {
+            Properties props = new Properties();
+            props.putAll(overrides);
+            Admin admin = Admin.create(clusterTestKit.newClientPropertiesBuilder(props).build());
             admins.add(admin);
```

Comment:
could you please remove this collection and move this implementation to interface?

Reply:
t seems that this collection is used in the close method. Would we lose the benefit of auto-closing the admin resource in the close method?
```
        @Override
        public void stop() {
            if (stopped.compareAndSet(false, true)) {
                admins.forEach(admin -> Utils.closeQuietly(admin, "admin"));
                admins.clear();
                Utils.closeQuietly(clusterTestKit, "cluster");
            }
        }
```

Reply:
We should encourage caller to close it explicitly.